src/cli/index.js

```diff
@@ -75,6 +75,10 @@ if (!appPath) {
 
 const optionsFromArguments = getOptionsFromArguments(options.rawArgs);
 const appName = optionsFromArguments.name || path.basename(appPath);
+const attributesToDisplay = (optionsFromArguments.attributesToDisplay || '')
+  .split(',')
+  .filter(Boolean)
+  .map(x => x.trim());
```

Comment:
technically an attribute could end/start in spaces no? Probably not a real issue

Reply:
Let's say you run the command:

```
npx create-instantsearch-app my-app \
  --attributes-to-display "name,description, location"
```

You will get the following attributes: `['name', 'description', ' location']`. It will then inject this into your template:

```javascript
_highlightResult. location.value // → breaks because of the space
```

If you trim the values, you'd get: `['name', 'description', 'location']`. This is the reason why we need this check.

Reply:
yes, knew why there could be valid padding, was just wondering if an attribute could start/end with spaces 🤔

Reply:
I can't see any reasons why. Do you?

Reply:
Now we know that we may have a bug here. Since it's rather an edge case, we can move forward.